### PR TITLE
add site-wide notification

### PR DESCRIPTION
**Pull request by umarmughal824**
#### What are the relevant tickets?
https://trello.com/c/p0Wds42c/135-site-wide-notifications

#### What's this PR do?
Add site wide notification in the cms

#### How should this be manually tested?
Add side notification and inspect the code in your page source.

### CODE REVIEW

**Comment by gsidebo** [comment]
Hey Umar. I mentioned in Slack that we won't need this notification, but that wasn't accurate. We do want to be able to allow admins to create a site-wide notification in the CMS, but we do not want to use the xpro approach of rendering it in its own element outside of the React app. For now, let's limit this PR to putting a hidden notification in the Django template. That means you can get rid of all of the Javascript. The notification will be rendered in the React app. I'll create a separate issue for that

File: main/templates/partials/notifications.html [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<div class="notifications site-wide d-none">
+  {% if notification %}
+    <div class="notification" data-notification-id="{{ notification.id }}">
+      <a href="#" class="close-notification"></a>
```
Comment (gsidebo) [inline main/templates/partials/notifications.html:4]: We can get rid of this element. We'll only be using this element to extract the HTML and show it in the React component, which will have its own close button

File: main/templates/partials/notifications.html [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<div class="notifications site-wide d-none">
```
Comment (gsidebo) [inline main/templates/partials/notifications.html:1]: I know this is probably copied over from xpro, but I'm not sure we need this outer element. We only render a single notification (or none at all). We can probably just keep the element inside the `if` tag and move the `site-wide d-none` class names down to that element

File: main/templates/partials/notifications.html (hunk 1 above)
Comment (umarmughal824) [reply main/templates/partials/notifications.html:4]: ok

File: main/templates/partials/notifications.html (hunk 2 above)
Comment (umarmughal824) [reply main/templates/partials/notifications.html:1]: ok

**Pull request merged**